src/modules/infobox/components/ImportContent.vue

```diff
@@ -323,3 +323,4 @@ export default {
     right: 50px;
 }
 </style>
+../utils/external-layer-providers.js
```

Comment:
I'm not sure this is wanted? If yes : what does it achieve?

Reply:
Oops you are right this is not intended, I'll remove that.

Reply:
done